fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -47,8 +49,10 @@ static_assert((ROCKSDB_MAJOR == 7 && ROCKSDB_MINOR == 7 && ROCKSDB_PATCH == 3),
 
 const std::string rocksDataFolderSuffix = "-data";
 const std::string METADATA_SHARD_ID = "kvs-metadata";
+const std::string SPECIAL_KEYS_SHARD_ID = "default";
```

Comment:
nit: suggest name it as DEFAULT_CF_NAME or similar, since special keys may not always map to the default CF.